src/pages/settings.tsx

```diff
@@ -8,4 +9,6 @@ const Settings = () => {
   );
 };
 
-export default Settings;
+const protectedSettings = requiresAdminAuth(Settings);
```

Comment:
This page still has the admin auth, but I'm not sure if settings will be for admins only? I thought settings were for any user.

Reply:
Since each user should only have access to their own settings, and it is currently simply a skeleton/plceholder file, I am happy to simplify this for now and let adding any future potential protections for a future PR, once we have some settings to put in there. ;)